<launch>
    <param name="use_sim_time" value="false"/>
    <node pkg="tf" type="static_transform_publisher" name="base_to_velodyne" args="0.0 0.0 0.0 0.0 0.0 0.0 base_link velodyne 100"/>
    <node name="encode" pkg="encode" type="encode" output="screen">
        <param name="base_frame"        value="base_link"/>
        <param name="map_frame"        value="map"/>
        <param name="min_scan_distance"        value="1.0"/>
        <param name="max_scan_distance"        value="40.0"/>
        <param name="point_cloud_topic"        value="velodyne_points"/>
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find encode)/launch/rviz.rviz" />
</launch>
